### DISCUSSION TITLE
Systemvariablen - Welchen Service für reelle bzw. Prozentzahlen

### DISCUSSION THREAD

**Discussion by ralan1965** [thkl/hap-homematic#486] [Q&A]
Hi,
vorweg, danke für das tolle Addon. 
Welchen Service muss ich wie für die Anzeige von Systemvariablen, Zahlen oder Prozente, auswählen. Ich bekomme zwar bei all meinen Versuchen ein Gerät ins HomeKit, aber entweder bleiben dieses ausgeschaltet oder es wird nicht unterstützt und zwar mit oder ohne Triggerprogramm.
Ich möchte mir gerne die Öffnungswerte der in meiner Homematic angelegten Shelly-Rolladenaktoren anzeigen lassen. Diese werden bei mir in Systemvariablen geschrieben.
<img width="804" alt="Bildschirmfoto 2021-01-10 um 16 19 21" src="https://user-images.githubusercontent.com/77229960/104634184-976a1500-56a0-11eb-98ae-26eaa735fbc5.png">

Sonst habe ich bisher alles hinbekommen :)

**Comment by ChrWi** [comment]
Ich habe derartige Systemvariablen wie folgt eingebunden:
<img width="788" alt="Bildschirmfoto 2021-09-30 um 20 31 25" src="https://user-images.githubusercontent.com/52505025/135511237-91c9ea77-afbd-4299-8896-3f1dcc2a0def.png">
Das mit Bezug zu den Rollladenaktoren anzuzeigen dürfte m.E. für die Shellys nicht gehen.
